kedro-org/kedro-viz
# Release v6.3.2

## Description

It's a patch version, since what's been merged doesn't warrant a minor or major version bump.

Here's what's in the release notes. Let me know if I missed anything.

#### Bug fixes and other changes

- Fix bug in layout for large pipeline warning message. (#1428)
- Fix bug where flowchart wasn't updating on back/forward clicks after selecting a registered pipeline. (#1422)

## QA notes

Take a look on Gitpod and run everything locally to make sure nothing is breaking.

## Checklist



<a href="https://gitpod.io/#https://github.com/kedro-org/kedro-viz/pull/1437"><img src="https://gitpod.io/button/open-in-gitpod.svg"/></a>
